### ISSUE TITLE
format checker skipping first token in file

### ISSUE THREAD

**Issue by brycepg** [PyCQA/pylint#2281]
Running `python -m pylint file.py --disable=all --enable=bad-continuation` on

```python
None
if None(
    'bar',
):
    pass
```
produces a `bad-continuation` message but 

```python
if None(
    'bar',
):
    pass
```

does not
